apiserver/allfacades.go

```diff
@@ -132,7 +132,7 @@ func AllFacades() *facade.Registry {
 	reg("Bundle", 1, bundle.NewFacade)
 	reg("CharmRevisionUpdater", 2, charmrevisionupdater.NewCharmRevisionUpdaterAPI)
 	reg("Charms", 2, charms.NewFacade)
-	reg("Cleaner", 2, cleaner.NewCleanerAPI)
+	reg("Cleaner", 2, cleaner.NewFacade)
```

Comment:
This lack of consistency really annoys me too :)  I suspect it happened b/c before, registration happened in separate packages. Now it's a lot more obvious.